pyccl/core.py

```diff
@@ -194,16 +189,10 @@
 
 # Configuration types
 transfer_function_types = {
-    'none':             lib.none,
-    'emulator':         lib.emulator,
-    'fitting_function': lib.fitting_function,
+    None:               lib.transfer_none,
     'eisenstein_hu':    lib.eisenstein_hu,
     'bbks':             lib.bbks,
-    'boltzmann':        lib.boltzmann,
-    'boltzmann_camb':   lib.boltzmann_camb,
-    'camb':             lib.boltzmann_camb,
     'boltzmann_class':  lib.boltzmann_class,
-    'class':            lib.boltzmann_class,
 }
 
 matter_power_spectrum_types = {
```

Comment:
I’m ok with this. Physically it is closer to reality maybe?

Reply:
Alex's comment below is related to this one. I suggest we rename `transfer` and `matter_power_spectrum` as he proposed.

Reply:
Alex agreed not to and this is a *ton* of work for marginal gains. If we want to do this, I suggest someone else do it in another PR. This would involve changes in nearly every piece of code in the repo.